docs/website/docs/building-from-source/riscv.md

```diff
@@ -0,0 +1,137 @@
+# RISC-V cross-compilation
+
+Running on a platform like RISC-V involves cross-compiling from a _host_
+platform (e.g. Linux) to a _target_ platform (a specific RISC-V CPU architecture
+and operating system):
+
+* IREE's _compiler_ is built on the host and is used there to generate modules
+  for the target
+* IREE's _runtime_ is built on the host for the target. The runtime is then
+  pushed to the target to run natively.
+
+## Prerequisites
+
+### Host environment setup
+
+You should already be able to build IREE from source on your host platform.
+Please make sure you have followed the [getting started](./getting-started.md)
+steps.
+
+### Install RISC-V cross-compile toolchain and emulator
+
+You'll need a RISC-V LLVM compilation toolchain and a RISC-V enabled QEMU
+emulator.
+
+* RISC-V toolchain is built from https://github.com/llvm/llvm-project (main branch).<br>
+  * Currently, the LLVM compiler is built on GNU toolchain, including libgcc,
+    GNU linker, and C libraries. You need to build GNU toolchain first.<br>
+  * Clone GNU toolchain from: https://github.com/riscv/riscv-gnu-toolchain
+    (master branch). Switch the "riscv-binutils" submodule to `rvv-1.0.x-zfh`
+    branch manually.
+* RISC-V QEMU is built from https://github.com/sifive/qemu/tree/v5.2.0-rvv-rvb-zfh
+
+An environment variable `RISCV_TOOLCHAIN_ROOT` needs
+to be set to the root directory of the installed GNU toolchain. The variable can
+be used in building the RISCV target and a LLVM AOT module.
+
+### Install Prebuilt RISC-V Tools (RISC-V 64-bit Linux toolchain)
+
+Execute the following script to download the prebuilt RISC-V toolchain and QEMU:
+
+```shell
+# In IREE source root
+$ ./build_tools/riscv/riscv_bootstrap.sh
+```
+**NOTE**:
+* You also need to set `RISCV_TOOLCHAIN_ROOT`
+(default at ${HOME}/riscv/toolchain/clang/linux/RISCV).
+
+## Configure and build
+
+### Host configuration
+
+Build and install on your host machine:
+
+``` shell
+cmake -B ../iree-build/ \
+  -DCMAKE_INSTALL_PREFIX=../iree-build/install \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  .
+cmake --build ../iree-build/ --target install
+```
+
+### Target configuration
+
+Currently IREE verifies (and tests in CI) two RISC-V targets:
```

Comment:
I'd like to be a bit careful with this sort of wording.

* "Currently ... two ... targets" in docs can get out of date with code easily
* I don't think we need to prove (in user documentation) how we achieve some technical capability. I'd rather just list which configurations are supported directly, perhaps with a list of known supported/not-yet-supported configurations like https://google.github.io/iree/#support-matrix (using https://squidfunk.github.io/mkdocs-material/reference/lists/#using-tasklists)

Reply:
Remove the "currently" and only note the instruction is for the RV64 Linux target